docs/CONTRIBUTING.md

```diff
@@ -15,6 +15,23 @@ You generally only need to submit a CLA once, so if you've already submitted one
 (even if it was for a different project), you probably don't need to do it
 again.
 
+## RAD Lab Repository
+
+We love and encourage your contributions to RAD Lab repo. Follow below steps to contribute on `features` and `bug fixes` for RAD Lab:
```

Comment:
We love and encourage your contributions to the RAD Lab repo. Follow the steps below to contribute on `features` and `bug fixes` for RAD Lab: